### ISSUE TITLE
Crash on load (NullPointerException at itemStack)

### ISSUE THREAD

**Issue by bondgb** [WhiskyTangoFawks/Expedition#6]
Log: http://pastebin.com/UTQ69B1M

**Comment by WhiskyTangoFawks** [comment]
Based on the log, I suspect that this is a mod incompatibility issue.  I'll upload a new beta to curseforge, it won't fix the issue, but it'll print out what item is causing the error when my mod looks at it.

**Comment by bondgb** [comment]
Hm, I don't see a descriptive new error with the new version. I DO get stuff like this in latest.log: 

[01:41:23] [Client thread/INFO] [STDOUT/wtfcore]: [wtf.crafting.RecipeParser:init:62]: Unsupported recipe type for class com.rwtema.extrautils2.crafting.EnchantRecipe
[01:41:23] [Client thread/INFO] [STDOUT/wtfcore]: [wtf.crafting.RecipeParser:init:62]: Unsupported recipe type for class com.rwtema.extrautils2.crafting.EnchantRecipe

[01:41:23] [Client thread/INFO] [STDOUT/wtfcore]: [wtf.crafting.RecipeParser:init:62]: Unsupported recipe type for class dan200.computercraft.shared.media.recipes.DiskRecipe

Basically for all mods that add their own crafting system. 

It's odd though because I disabled everything but world generation modules in the config.

**Comment by WhiskyTangoFawks** [comment]
try beta 11.4

**Comment by bondgb** [comment]
It works!
